docs/faq.md

```diff
@@ -0,0 +1,56 @@
+# FAQ
+
+## Can I run my own instance?
+
+Yes! We host a public canonical version of the Matrix Public Archive at
+[archive.matrix.org](https://archive.matrix.org/) that everyone can use but feel free to
+also run your own instance (setup instructions in the [readme](../README.md)).
+
+## How do I provide via servers to join from?
```

Comment:
Are "via servers" a thing in Matrix? I'm confused by this question. 🤔

Reply:
If your server doesn't know about a given room ID, then it won't be able to join it (room ID's aren't routable on their own). Which is where "via" servers come in and you provide a server that already knows about the room to join through. See the [`server_name` query parameter with the `/join` endpoint](https://spec.matrix.org/v1.5/client-server-api/#post_matrixclientv3joinroomidoralias).

Maybe this detail doesn't help much in reality. I can remove it for now until I can phrase it in a useful way.